__tests__/app.test.js

```diff
@@ -0,0 +1,60 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+
+//SETUP AND SHUTDOWN
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+// TEST SUITE
+describe("/api/topics", () => {
+  describe("GET", () => {
+    test("200: Should recieve a 200 code for a sucessful request", () => {
+      return request(app).get("/api/topics").expect(200);
+    });
+
+    test('200: Should respond with a body of topic objects in an array, with "slug" and "description properties"', () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
```

Comment:
This line is testing for the 200 so the first test is superfluous. This test however is great.